flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue

```diff
@@ -103,14 +142,14 @@ export default {
        
   methods: {
     yut () {
-      console.log(this.$store.state.Variables[8]);
-         console.log(this.$store.state.Variables[8].rainfall.transform.data_orig);
+      console.log(this.$store.state.RothC_config.Variables[12]);
+         console.log(this.$store.state.RothC_config.Variables[12].soilCover.transform.data_orig);
       console.log("Sss");
     },
-    foobar (event) {
+    // foobar (event) {
     
-         console.log(event.target.value);
-        console.log(this.$store.state.Variables[8].rainfall.transform.data_orig);
+    //      console.log(event.target.value);
+    //     console.log(this.$store.state.RothC_config.Variables[12].soilCover.transform.data_orig);
```

Comment:
Is this dead code?

Reply:
Hi @aornugent, yes I think the code needs more cleaning...